src/java.net.http/share/classes/jdk/internal/net/http/common/SSLFlowDelegate.java

```diff
@@ -781,6 +781,7 @@ private void processData() {
                                 + hsTriggered() + ", needWrap:" + needWrap());
 
                 while (Utils.synchronizedRemaining(writeList) > 0 || hsTriggered() || needWrap()) {
+                    if (scheduler.isStopped()) return;
```

Comment:
If the `scheduler` is stopped then would this task instance be ever called again? If it won't be called again, then do you think we should perhaps drain the queued `writeList` to reduce any memory resource accumulation till the next GC cycle?